Patch to add StopTimers method, which will stop the tick and dwell timers.
Used by Komodo to ensure the timers are disabled when a Scintilla view gets
closed and cleaned up, bug 82032.

Index: include/Scintilla.h
===================================================================
--- include/Scintilla.h	(revision 27050)
+++ include/Scintilla.h	(working copy)
@@ -650,6 +650,7 @@
 #define SC_ALPHA_NOALPHA 256
 #define SCI_SETCARETLINEBACKALPHA 2470
 #define SCI_GETCARETLINEBACKALPHA 2471
+#define SCI_STOPTIMERS 2472
 #define CARETSTYLE_INVISIBLE 0
 #define CARETSTYLE_LINE 1
 #define CARETSTYLE_BLOCK 2
Index: include/Scintilla.iface
===================================================================
--- include/Scintilla.iface	(revision 27050)
+++ include/Scintilla.iface	(working copy)
@@ -1750,6 +1750,9 @@
 # Get the background alpha of the caret line.
 get int GetCaretLineBackAlpha=2471(,)
 
+# Stop all Scintilla timers (caret, dwell, ...).
+fun void StopTimers=2475(,)
+
 enu CaretStyle=CARETSTYLE_
 val CARETSTYLE_INVISIBLE=0
 val CARETSTYLE_LINE=1
